treeopt/treeOpt.py

```diff
@@ -262,11 +228,25 @@ def write_data(self, npArray, filename):
         :return: Nothing
         :rtype: None
         """
-
-        Path(os.getcwd() + "/treeOptData").mkdir(parents=True, exist_ok=True)
-        path = os.path.join(os.getcwd() + "/treeOptData", filename + ".csv")
-        np.savetxt(path, npArray, delimiter=",")
-
+        
+        # Path(os.getcwd() + os.sep + "treeOptData").mkdir(parents=True, exist_ok=True)
+        # path = os.path.join(os.getcwd() + os.sep + "treeOptData", filename + ".csv")
+        # np.savetxt(path, npArray, delimiter=",")
+        
+        import time
+        
+        Path(filepath).mkdir(parents=True, exist_ok=True)
+        path = os.path.join(filepath, filename + ".csv")
+        np.savetxt(path, Data, delimiter=",")
+        
+        time.sleep(2)
```

Comment:
Why do we need this?

Reply:
This was there to test something. I have forgotten to delete it.